### PR TITLE
#184948278 Buyer view recommended/similar products 

### PR DESCRIPTION
**Pull request by GasanaJr**
#### What does this PR do?
This PR adds recommended/similar products view on the front end.
#### Description of Task to be completed?
- adds similar items view on the front based on what product user searches  
#### How should this be manually tested?
- clone this repo `git clone https://github.com/atlp-rwanda/e-comm-team-amigos-fn`
- switch your working directory `cd e-comm-team-amigos-fn`
- create an `.env` and update it with `.env-example`
- install all dependencies by running `npm install`
- in your terminal run `npm start`
#### Any background context you want to provide?
- Node JS installed in your local machine
#### Screenshots
-Screenshot 2023-05-17 at 3 01 30 AM
-Screenshot 2023-05-17 at 3 00 38 AM
- Screenshot 2023-05-17 at 11 41 47 AM
- Screenshot 2023-05-18 at 8 52 16 AM
#### What are the relevant pivotal tracker stories?
Story ID: [#184948278](https://www.pivotaltracker.com/story/show/184948278)

### CODE REVIEW

File: src/components/RelatedItems/related.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    background-color: #FFFFFF;
+    color: #000000;
+    border: 0.3px solid gray;
+}
+
+.relatedItems {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    //justify-content: center;
```
Comment (Emile-SHUMBUSHO) [inline src/components/RelatedItems/related.scss]: commented code should not be pushed to the remote repository

Comment (GasanaJr) [reply src/components/RelatedItems/related.scss]: Thanks for the reminder bro. It is still a work in progress. I will remove them.

**Comment by Emile-SHUMBUSHO** [comment]
Add some margin-bottom and add footer if it's possible
Screenshot 2023-05-18 at 08 52 16

**Review by Emile-SHUMBUSHO — APPROVED** [review]

**Review by AbdulKhaliq59 — APPROVED** [review]
I think I may suggest two things:
-  to add border-radius on this card 
-  remove console.log()
and unless it `LGTM`
Screenshot 2023-05-18 at 09 53 14

**Comment by GasanaJr** [comment]
> I think I may suggest two things:
> 
> * to add border-radius on this card
> * remove console.log()
>   and unless it `LGTM`
> 
> Screenshot 2023-05-18 at 09 53 14

The issue is resolved now @AbdulKhaliq59

**Review by Rwibutso — APPROVED** [review]

File: src/components/Product.jsx [hunk 2]
```diff
@@ -33,136 +37,236 @@ const ProductDetails = () => {
...
+					method: "POST",
+					mode: "cors",
+				},
+			);
+			const related_items = await related.json();
+			setRelated(related_items.products);
+		};
+		getRelatedProducts();
+	}, [category]);
+
```
Comment (Habinezajanvier) [inline src/components/Product.jsx]: Can we please make sure that we are handling request in redux actions

File: src/components/Product.jsx [hunk 3]
```diff
@@ -33,136 +37,236 @@ const ProductDetails = () => {
...
+													product.images[0],
+												)
+											}
+										>
+											<img
+												src={product.images[0]}
+												alt="images"
+											/>
+										</div>
+									</>
```
Comment (Habinezajanvier) [inline src/components/Product.jsx]: Starting from line 73 till here can be wrapped in a array.map functionality hence reducing lines of codes

**Review by Habinezajanvier — CHANGES_REQUESTED** [review]

**Review by Habinezajanvier — APPROVED** [review]
👍

**Pull request merged**